_data/hosting_providers.json

```
@@ -2568,5 +2578,5 @@
     "plan": "",
     "reviewed": "2021.10.09",
     "note": ""
-  }
+  },
```

Please remove this trailing comma.

```suggestion
  }
```